sdk/nodejs/__tests__/client.spec.ts

```diff
@@ -18,3 +18,34 @@ describe('DVCClient', () => {
         })
     })
 })
+
+describe('variable', () => {
+    const user = {
+        user_id: 'node_sdk_test',
+        country: 'CA'
+    }
+    const expectedUser = expect.objectContaining({
+        user_id: 'node_sdk_test',
+        country: 'CA'
+    })
+    
+    describe('variableDefaulted event', () => {
+        test('does not get sent if variable is in bucketed config',async () => {
```

Comment:
you should add another test case for "it does get sent when the variable is not in the bucketed config"

Reply:
oh i did initially, but that test also passed w the previous code, so i removed it.
should i make it so that it fails w the prev code? or is it ok if it passes there too?

Reply:
it would have passed before as well because the old behaviour would still have sent `variableDefaulted` if the variable wasn't in the config. The bug you fixed was just for the case where it _was_ in the config. However, there isn't an existing test to cover the fact that variableDefaulted is being sent, and since you changed the code that sends it it would be a good idea to add one to make sure that still works.